### PR TITLE
Fix deep buffer playback pipelines for cavs

### PR DESCRIPTION
**Pull request by ranj063**
APL cannot support 2 deep buffer playback PCM's. So fix this by adding only 1deep buffer playback PCM on SSP5 for APL and SSP0 for the rest

### CODE REVIEW

**Comment by ranj063** [comment]
fixes https://github.com/thesofproject/sof/issues/4526

File: tools/topology/topology1/sof-cavs-nocodec.m4 [hunk 1]
```diff
@@ -107,10 +107,13 @@ dnl     time_domain, sched_comp)
 
 # Low Latency playback pipeline 1 on PCM 0 using max 2 channels of PIPE_BITS.
 # Set 1000us deadline on core 2 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+# For BXT, this will be a regular volume playback pipeline. For all other cavs platforms,
+# this will be replaced by mixer based playback pipeline
```
Comment (plbossart) [inline tools/topology/topology1/sof-cavs-nocodec.m4:111]: why special-case? it'd be fine to use a mixer on all 3 dai pipelines, and only connect a second host pipeline for one of the mixers.
That way all 3 playback path have the same latency/processing/settings.

Comment (plbossart) [reply tools/topology/topology1/sof-cavs-nocodec.m4:111]: that would also make the pipeline management less complicated with fewer options.

**Review by plbossart — COMMENTED** [review]
Wondering if we can simplify by keeping the structure identical instead of special casing.

File: tools/topology/topology1/sof-cavs-nocodec.m4 (hunk 1 above)
Comment (ranj063) [reply tools/topology/topology1/sof-cavs-nocodec.m4]: @plbossart fixed now. here's  how the tplg's look now 
APL:
tplg
Rest:
tplg

Comment (plbossart) [reply tools/topology/topology1/sof-cavs-nocodec.m4]: just what the doctor ordered. Thanks Ranjani.

**Review by plbossart — APPROVED** [review]
LGTM @ranj063 
The daily tests start in 6mn, probably too late to merge but can we start a daily test later this evening with this PR, so that we can merge with confidence tomorrow before the week-end?

**Comment by ranj063** [comment]
> LGTM @ranj063
> The daily tests start in 6mn, probably too late to merge but can we start a daily test later this evening with this PR, so that we can merge with confidence tomorrow before the week-end?

Yes, will do!

**Review by lgirdwood — APPROVED** [review]

**Pull request merged**